docs/testing.md

```diff
@@ -0,0 +1,460 @@
+# Testing
+
+This guide covers testing connect-python services and clients.
+
+## Setup
+
+Install httpx for testing connect-python services and clients:
```

Comment:
I don't know if we need this, won't it always be installed as a dependency of connect-python?

Reply:
Thanks for catching that! Fixed in 58d6653.